### PR TITLE
Wrap it all up as a deployable controller.

### PR DESCRIPTION
**Pull request by mattmoor**
This ties together all of the prior changes into a deployable controller that runs alongside a knative/serving installation.

`ko apply -f config` will deploy the controller, which can then be configured by changing the `certificate.class` in `config-network` to `net-http01.certificate.networking.knative.dev`.

### CODE REVIEW

File: cmd/controller/main.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/injection/sharedmain"
+	"knative.dev/pkg/signals"
+
+	"knative.dev/net-http01/pkg/challenger"
+	"knative.dev/net-http01/pkg/reconciler/certificate"
+)
+
+func main() {
+	// Uncomment this to use the Let's Encrypt Staging environment.
```
Comment (tcnghia) [inline cmd/controller/main.go]: Turn this into flag?

Comment (mattmoor) [reply cmd/controller/main.go]: I think I'd rather move it into a configmap in follow-up?

Comment (tcnghia) [reply cmd/controller/main.go]: sgtm

Comment (mattmoor) [reply cmd/controller/main.go]: Opened: https://github.com/knative/net-http01/issues/10

**Pull request merged**